Libs/DICOM/Widgets/Testing/Cpp/CMakeLists.txt

```diff
@@ -12,11 +12,20 @@ create_test_sourcelist(Tests ${KIT}CppTests.cpp
   ctkDICOMModelTest2.cpp
   ctkDICOMObjectModelTest1.cpp
   ctkDICOMQueryResultsTabWidgetTest1.cpp
-  ctkDICOMQueryRetrieveWidgetTest1.cpp
   ctkDICOMServerNodeWidgetTest1.cpp
   ctkDICOMThumbnailListWidgetTest1.cpp
   )
 
+if(CTK_QT_VERSION VERSION_GREATER "4")
```

Comment:
The check should be done based on enabled/disabled features. We should avoid features disappearing silently based on detected Qt version.